core/src/test/java/com/netflix/conductor/core/utils/ExternalPayloadStorageUtilsTest.java

```diff
@@ -86,18 +87,36 @@ public void setup() {
 
     @Test
     public void testDownloadPayload() throws IOException {
+        String path = "key1";
+        Map<String, Object> db = new HashMap<>();
+        db.put("key1", "value1");
+        db.put("key2", 200);
+        byte[] payloadBytes = objectMapper.writeValueAsString(Collections.singletonMap("key1", "value1")).getBytes();
+        when(externalPayloadStorage.download(path))
+                .thenReturn(new ByteArrayInputStream(payloadBytes));
+
+        Map<String, Object> result = externalPayloadStorageUtils.downloadPayload(path);
+        assertNotNull(result);
+        assertEquals(Collections.singletonMap("key1", "value1"), result);
+    }
+
+    @Test
+    public void testDownloadNonExistentPayload() {
         String path = "test/payload";
 
-        Map<String, Object> payload = new HashMap<>();
-        payload.put("key1", "value1");
-        payload.put("key2", 200);
-        byte[] payloadBytes = objectMapper.writeValueAsString(payload).getBytes();
+        Map<String, Object> db = new HashMap<>();
+        db.put("key1", "value1");
+        db.put("key2", 200);
+
+        String errorMessage =
```

Comment:
this is a constant in PostgresPayloadStorage, don't copy it here, reuse the same constant

Reply:
Yes you're right, I moved this parameter to the class constructor